railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -47,6 +47,9 @@ Rails.application.configure do
   # when problems arise.
   config.log_level = :debug
 
+  # Enforces that all of the following environment variables are defined before server start
+  # config.required_env_vars = ['API_KEY', 'OTHER_API_KEY']
```

Comment:
@dhh what do you think about this being included directly on production.rb? I can see how this feature can be useful but I don't like the idea of promoting it by default. In my projects we usually don't use any ENV key.